README.md

```diff
@@ -7,38 +7,31 @@ A __Rust__ binding for the [GSL library](http://www.gnu.org/software/gsl/) (the
 
 This binding requires the [GSL library](http://www.gnu.org/software/gsl/) library to be installed.
 
-To build it, please use :
+This crate works with Cargo and is on
+[crates.io](https://crates.io/crates/GSL).
+Just add the following to your `Cargo.toml` file:
 
-```Shell
-> make
+```toml
+[dependencies]
+GSL = "*"
 ```
 
-This command builds __rgsl__, the examples and the documentation.
-
-You can build them separatly too.
+Add the following line to your source:
```

Comment:
"source code" instead of "source" maybe ? (I'm not an english native so correct me if I'm wrong !)